workflows/references/main.nf

```diff
@@ -306,57 +306,70 @@ workflow REFERENCES {
     versions = versions.mix(TABIX_KNOWN_INDELS.out.versions)
     versions = versions.mix(TABIX_KNOWN_SNPS.out.versions)
 
+    // ch_bwamem1 >> '<SPECIES>/<SOURCE>/<GENOME>/Sequence/BWAIndex'
```

Comment:
Remove if not used anymore

Reply:
that was never used, it's a comment for me, regarding what I intend to implement

Reply:
cf https://github.com/nf-core/references/pull/41#discussion_r1862156213